src/vs/workbench/contrib/chat/browser/chatInputPart.ts

```diff
@@ -775,6 +775,23 @@ export class ChatInputPart extends Disposable implements IHistoryNavigationWidge
 					}
 					this.openerService.open(file, options);
 				}));
+
+				store.add(dom.addDisposableListener(widget, dom.EventType.KEY_DOWN, (e: KeyboardEvent) => {
+					const event = new StandardKeyboardEvent(e);
+					if (event.equals(KeyCode.Enter)) {
+						dom.EventHelper.stop(e, true);
+						const options: Mutable<OpenInternalOptions> = {
+							fromUserGesture: true
+						};
+						if (range) {
+							// eslint-disable-next-line local/code-no-dangerous-type-assertions
+							options.editorOptions = {
+								selection: range
+							} as ITextEditorOptions as any;
```

Comment:
This was similar to another place in the code. It feels like a hack with the cast, is there a proper way to do this?